content/docs/jsx-in-depth.md

```diff
@@ -331,39 +331,39 @@ You can mix together different types of children, so you can use string literals
 </div>
 ```
 
-A React component can also return an array of elements:
+Sebuah komponen React juga bisa memberi hasil balikan (*return*) berupa sebuah *array* elemen:
 
 ```js
 render() {
-  // No need to wrap list items in an extra element!
+  // Tidak perlu untuk membungkus daftar itemd dalam sebuah elemen tambahan!
   return [
-    // Don't forget the keys :)
-    <li key="A">First item</li>,
-    <li key="B">Second item</li>,
-    <li key="C">Third item</li>,
+    // Jangan lupa kuncinya :)
+    <li key="A">Item pertama</li>,
+    <li key="B">Item kedua</li>,
+    <li key="C">Item ketiga</li>,
   ];
 }
 ```
 
-### JavaScript Expressions as Children {#javascript-expressions-as-children}
+### Ekspresi JavaScript as Turunan (*Children*) {#javascript-expressions-as-children}
```

Comment:
```suggestion
### Ekspresi JavaScript sebagai *Children* {#javascript-expressions-as-children}
```

You missed a word here. Also keep *children* untranslated.

Reply:
>### Ekspresi JavaScript sebagai *Children* {#javascript-expressions-as-children}

Thank you, your correction is applied